lib/rdstation/authentication.rb

```diff
@@ -47,6 +47,21 @@ def update_access_token(refresh_token)
       ApiResponse.build(response)
     end
 
+    def self.revoke(access_token:)
```

Comment:
I have a couple of suggestions for this method:

1. Could you put the URL into a constant?
2. The headers construction might be moved to a different method.
3. The body construction might be moved to a different method.

Thus we can reach something like this:

```ruby
response = self.post(REVOKE_URL, body: body, headers: headers)
```